packages/core/src/render3/util.ts

```diff
@@ -127,7 +127,8 @@ export function isComponentDef<T>(def: DirectiveDef<T>): def is ComponentDef<T>
   return (def as ComponentDef<T>).template !== null;
 }
 
-export function isLContainer(value: RElement | RComment | LContainer | StylingContext): boolean {
+export function isLContainer(
+    value: RElement | RComment | LContainer | LView | StylingContext | null): boolean {
```

Comment:
We can let TypeScript narrow the type of `value` after this call, would avoid some casts :)

```suggestion
    value: RElement | RComment | LContainer | LView | StylingContext | null): value is LContainer {
```